templates/checkout/order-confirmation.tpl

```diff
@@ -0,0 +1,30 @@
+{extends "page.tpl"}
+
+{block name="page_title"}
+  {l s='Order confirmation'}
+{/block}
+
+{block name="page_content_container" prepend}
+  <section id="content-hook_payment_return">
```

Comment:
Well, the only reason was to separate "content" and "hook_payment_return"... I've been seeing this with a question mark too, but was sure this would been commented on and decide final then :)